### ISSUE TITLE
Error in loss

### ISSUE THREAD

**Issue by ChristofferEdlund** [kozistr/ESRGAN-tensorflow#4]
https://github.com/kozistr/ESRGAN-tensorflow/blob/b980f5e856e663be4115a6883c69d8c9b7e66b8e/losses.py#L8-L11

```
            real = real - tf.reduce_mean(fake)
            fake = fake - tf.reduce_mean(real)
```

I presume that you want the original real variable and not the relativistic one when calculating fake?
